lib/wordpress.js

```diff
@@ -0,0 +1,48 @@
+var _ = require('underscore');
+var params = require('querystring').parse(window.location.search.split('?').pop());
+
+module.exports = {
```

Comment:
I did it this way to leave room for other functions here, like `.getProtectedPostView()` on line 44. Rather than repeating `module.exports` at various arbitrary points throughout the file, I think it's cleaner to do it this way.